vars/Debian.yml

```diff
@@ -1,3 +1,4 @@
 ---
 __unattended_origins_patterns:
-  - 'origin=Debian,archive=${distro_codename},label=Debian-Security'
+  - 'origin=Debian,archive=stable,label=Debian-Security'
+  - 'origin=Debian,archive=oldstable,label=Debian-Security'
```

Comment:
I was thinking in case that we do a dist-upgrade, ```oldstable``` would help to do not upgrade directly the package and keep the ```oldstable```.